src/sections/Feature.vue

```diff
@@ -43,6 +43,12 @@
           >
             <i>{{ season }} applications are now open!</i>
           </p>
+          <p
+            v-else
+            class="secondary"
+          >
+            <i> We are not currently accepting new applications. In the meantime, keep an eye on this page for updates!</i>
```

Comment:
```suggestion
            <i>Unfortunately we are not currently accepting new applications, but in the meantime, keep an eye on this website for updates!</i>
```

Reply:
would like some more opinions on the wording here before we merge too - while it's important to have the message, we don't want it to take away too much from the introductory aspect of the site